invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/search.html

```diff
@@ -14,98 +14,5 @@
 {%- endblock %}
 
 {%- block page_body %}
-
-<!-- TODO: Auto-generate for new API -->
-<div data-invenio-search-config='{
-  "aggs": [
-    {
-      "aggName": "access_status",
-      "field": "access.status",
-      "title": "{{_("Access status")}}"
-    },
-    {
-      "aggName": "resource_type",
-      "field": "resource_type.type",
-      "title": "{{_("Resource types")}}",
-      "childAgg": {
-        "aggName": "inner",
-        "field": "resource_type.subtype",
-        "title": "{{_("Resource types")}}"
-      }
-    },
-    {
-      "aggName": "languages",
-      "field": "languages",
-      "title": "{{_("Languages")}}"
-    },
-    {
-      "aggName": "subjects",
-      "field": "subjects",
-      "title": "{{_("Subjects")}}"
-    }
-  ],
-  "appId": "rdm-search",
-  "initialQueryState": {
-    "hiddenParams": null,
-    "size": 10,
-    "page": 1,
-    "sortBy": "bestmatch"
-  },
-  "layoutOptions": {
-    "gridView": false,
-    "listView": true
-  },
-  "paginationOptions": {
-    "defaultValue": 10,
-    "resultsPerPage": [
-      {
-        "text": "10",
-        "value": 10
-      },
-      {
-        "text": "20",
-        "value": 20
-      },
-      {
-        "text": "50",
-        "value": 50
-      }
-    ]
-  },
-  "searchApi": {
-    "axios": {
-      "headers": {
-        "Accept": "application/vnd.inveniordm.v1+json"
-      },
-      "url": "/api/records",
-      "withCredentials": true
-    },
-    "invenio": {
-      "requestSerializer": "InvenioRecordsResourcesRequestSerializer"
-    }
-  },
-  "sortOrderDisabled": true,
-  "defaultSortingOnEmptyQueryString": {
-    "sortBy": "newest"
-  },
-  "sortOptions": [
-    {
-      "sortBy": "bestmatch",
-      "text": "{{_("Best match")}}"
-    },
-    {
-      "sortBy": "newest",
-      "text": "{{_("Newest")}}"
-    },
-    {
-      "sortBy": "oldest",
-      "text": "{{_("Oldest")}}"
-    },
-    {
-      "sortBy": "version",
-      "text": "{{_("Version")}}"
-    }
-  ]
-}'></div>
-
+<div data-invenio-search-config='{{ search_app_config(app_id="rdm-search") | tojson }}'></div>
```

Comment:
`rdm-search`, this is *naming* and not from now... But what about `record-search`, since all are `rdm` search apps 😅